addon-resizer/nanny/kubernetes_client.go

```diff
@@ -48,12 +48,13 @@ const (
 )
 
 type kubernetesClient struct {
-	namespace  string
-	deployment string
-	pod        string
-	container  string
-	clientset  *kubernetes.Clientset
-	useMetrics bool
+	namespace   string
+	deployment  string
+	pod         string
+	container   string
+	clientset   *kubernetes.Clientset
+	useMetrics  bool
+	runOnMaster bool
```

Comment:
Which behavior are you mentioning about here?

Reply:
All the functionality gated by runOnMaster, I think it's not necessarily tied to the fact that it's running on the control-plane, but that it's not running as part of the same pod it's managing.
Aside from that I think the k8s community is trying to avoid the term master these days and is favoring "control-plane": https://github.com/kubernetes/website/issues/6525

Reply:
Makes sense. Thanks! Updated all "master" to "control plane".